src/main/java/com/minecolonies/rankup/util/CommonUtils.java

```diff
@@ -50,4 +66,39 @@ public static String timeDescript(int timeNeeded)
         }
         return msg.toString();
     }
+
+    public static String dateNow()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        java.util.Date today = Calendar.getInstance().getTime();
+        return dateFormat.format(today);
+    }
+
+    public static String dateFormat(final String inputDate)
+    {
+        final String date = inputDate.replace("/", "-");
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        SimpleDateFormat reverseFormat = new SimpleDateFormat("dd-MM-yyyy");
+        dateFormat.setLenient(false);
+        reverseFormat.setLenient(false);
+
+        try
+        {
+            dateFormat.parse(date);
+        }
+        catch (ParseException ex)
+        {
+            try
+            {
+                Date newDate = reverseFormat.parse(date);
+                return dateFormat.format(newDate);
+            }
+            catch (ParseException e)
+            {
+                ex.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1148)